community/decision-log.md

```diff
@@ -36,3 +37,40 @@ None.
 * Mike Bowen (BlackRock)
 * Mukulika Kapas (Intuit)
 * Shoubhik Bose (Red Hat)
+
+---
+## Argo Decision Making Process
+
+### Date
+2020 Sep 14
+
+### Context
+
+This documents the basic principles for decision making in the Argo Project, particularly by the bootstrap committee and in the future, by the steering and technical oversight committees, to promote transparency, community participation and better decision making.
+
+### Decisions and Rationales
+
+1) Decisions are made based on consensus (unanimous consent).
+
+Consensus promotes an inclusive consideration of all viewpoints, particularly those voiced by a minority of community members compared with alternative decision making processes such as voting. Concensus means that everone agrees to try a particular course of action, even though as an individuas, one may feel that this may not be the best choice (dissenting opionion).
```

Comment:
```suggestion
Consensus promotes an inclusive consideration of all viewpoints, particularly those voiced by a minority of community members compared with alternative decision making processes such as voting. Consensus means that everyone agrees to try a particular course of action, even though as an individuals, one may feel that this may not be the best choice (dissenting opinion).
```